week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,17 @@
+let btn = document.getElementById('btn')
+
+btn.addEventListener('click', function(){
+    let ulEle = document.getElementById('ul')
+    let liTag = document.createElement('li')
+    ulEle.appendChild(liTag)
+    let imgTag = document.createElement('img')
+    liTag.appendChild(imgTag)
+    
+fetch('https://dog.ceo/api/breeds/image/random')
+.then(function(response){
+ return response.json();
+})
+.then(function(dogImage){
+    imgTag.src = dogImage.message
+})
+});
```

Comment:
Good work, you could have separated your code and make it more understandable. clean Up you code only.